src/DB/db.js

```diff
@@ -0,0 +1,9 @@
+exports.getEvents = {
```

Comment:
I prefer naming folder in kebab-case, all letter in lowercase.

Reply:
doing that messes everything up for that particular file
![Screenshot (305)](https://user-images.githubusercontent.com/25851637/70971193-7310c300-20b1-11ea-9e82-ca77c7db0312.png)

Reply:
I meant the folder, not the variable i.e should be `src/db/db.js`